terraform/modules/task-definitions/router-api/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "image_tag" {
+  description = "Container Image Tag"
+  type        = string
+}
+
+variable "mesh_name" {
+  type = string
+}
+
+variable "mongodb_url" {
+  type = string
+}
+
+variable "execution_role_arn" {
+  type = string
+}
+
+variable "task_role_arn" {
+  type = string
+}
+
+variable "sentry_environment" {
+  type = string
+}
+
+variable "assume_role_arn" {
+  type        = string
+  description = "(optional) AWS IAM role to assume. Uses the role from the environment by default."
+  default     = null
+}
+
+variable "service_name" {
+  type = string
+}
+
+variable "router_urls" {
```

Comment:
These don't actually seem to be URLs. Maybe call this router_nodes? (avoids potential for confusion and keeps it consistent with the environment variable)

Reply:
Thinking a bit more broadly, we might want to change the way Router API discovers Router instances (or the way Router is notified of changes). A static list basically prevents us from scaling Router without mucking about with Router API. This is a good example of an app issue to add to the backlog.